### DISCUSSION TITLE
How to get data from trips api ?

### DISCUSSION THREAD

**Discussion by vineethktpla** [flobz/psa_car_controller#828] [Q&A]
I got these values when i called the trips api call after the update. 
Its very old data. How to get the latest one
![image](https://github.com/flobz/psa_car_controller/assets/146456863/d08aebf5-cbd6-4f50-9ed3-f8f1355d9229)
![image](https://github.com/flobz/psa_car_controller/assets/146456863/58c4c334-ef9c-40b3-ab6c-692fd9b0477a)

**Comment by zazazouthecat** [comment]
http://YOUR_IP_SERVER:5000/vehicles/trips

![image](https://github.com/flobz/psa_car_controller/assets/30299307/066d4b65-26f9-4a84-9a9e-43889ac53c78)

**Comment by vineethktpla** [comment]
I get only the first data from this api call via home assistant but then I found a way around to set json_attributes_path: $.[-1:] to get the last available data

**Reply by eerwinn** [reply]
I was looking for this. Thanks!
